wavefront/events.go

```diff
@@ -65,7 +66,36 @@ func (a *APIClient) createEvent(name string, annotations map[string]string, tags
 	return a.doEventRequest(req)
 }
 
-func (a *APIClient) EndOngoingEvent(eventID string) ([]byte, error) {
+func (a *APIClient) EndOngoingEvent(eventID string, eventJSON []byte, newAnnotations map[string]string) ([]byte, error) {
+	if eventJSON != nil && newAnnotations != nil {
```

Comment:
It might help readability if the contents of this `if` were moved into a func called `updateAnnotations()` or something.

Reply:
👍